### PR TITLE
updated base build config for smartsheets & an optional rowId link in…

### PR DESCRIPTION
**Pull request by matallen**
… the event log as an enhancement

### CODE REVIEW

File: .openshift/templates/ninja-board-build.yml [hunk 1]
```diff
@@ -37,10 +37,14 @@ objects:
...
         RUN yum clean all && \
-          INSTALL_PKGS="python-requests pytz" && \
+          yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+          INSTALL_PKGS="python2-pip python-requests pytz" && \
           yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
           rpm -V $INSTALL_PKGS && \
-          yum clean all
+          yum clean all && \
+          pip install --no-cache-dir enum smartsheet-python-sdk
+          pip install urllib3 --upgrade && \
```
Comment (sabre1041) [inline .openshift/templates/ninja-board-build.yml:46]: Can we use the ``--no-cache-dir` as well when using the other `pip install` commands

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Quick change. Otherwise good

File: .openshift/templates/ninja-board-build.yml (hunk 1 above)
Comment (matallen) [reply .openshift/templates/ninja-board-build.yml]: yes, makes sense - additional commit below to add '--no-cache-dir'

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**